common-npm-packages/azure-arm-rest-v2/azure-arm-aks-service.ts

```diff
@@ -30,8 +30,8 @@ export class AzureAksService {
 
     public beginRequest(uri: string,  parameters: {}) : Promise<webClient.WebResponse> {
          var webRequest = new webClient.WebRequest();
-         webRequest.method = 'GET';
-         webRequest.uri = this._client.getRequestUri(uri, parameters, null,'2017-08-31');
+         webRequest.method = 'POST';
```

Comment:
This will affect ALL requests that will be done using AzureAksService. Are you sure that there are no GET requests anymore?

Reply:
Unfortunately, there is no description for the rest api of previous versions in MS documentation. However, I checked azure-rest-api-spec repository.
Version 2017-08-31 had [accessProfile/{profileName} GET](https://github.com/Azure/azure-rest-api-specs/blob/f14f1d17370ed1d56f06d8933f434832b3f0a618/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2017-08-31/managedClusters.json#LL209C4-L209C4) (that was already marked as deprecated) and [accessProfiles/{roleName}/listCredential POST](https://github.com/Azure/azure-rest-api-specs/blob/f14f1d17370ed1d56f06d8933f434832b3f0a618/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2017-08-31/managedClusters.json#LL157C5-L157C182) (deprecated since [2020-04-01](https://github.com/Azure/azure-rest-api-specs/tree/f14f1d17370ed1d56f06d8933f434832b3f0a618/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2020-04-01))
ListClusterAdminCredentials and ListClusterUserCredentials in [2018-03-31](https://github.com/Azure/azure-rest-api-specs/blob/f14f1d17370ed1d56f06d8933f434832b3f0a618/specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2018-03-31/managedClusters.json#L237).
There is no GET requests anymore.